### ISSUE TITLE
update setup-python version

### ISSUE THREAD

**Issue by zzkt** [riscy/melpazoid#28]
The current version (at [d482d5d](https://github.com/riscy/melpazoid/commit/d482d5d3e9740f497ae4411adec384233de2b728)) didn't seem to work for me.

With an update to `actions/checkout` and `actions/setup-python` it looks like it's working as expected (and entered as an issue rather than PR since there may be something i'm overlooking)

```
    - uses: actions/checkout@v3
    - name: Set up Python 3.10
      uses: actions/setup-python@v4
      with:
        python-version: '3.10'
```

**Comment by riscy** [comment]
Thank you for this!  I've been falling a bit behind, but I'll aim to look this week.

**Comment by riscy** [comment]
Would you like to PR this so you get proper credit?

**Comment by zzkt** [comment]
...as PR at https://github.com/riscy/melpazoid/pull/29

**Comment by riscy** [comment]
Closed in #29
